frontend/src/txns/execute.ts

```diff
@@ -1,10 +1,93 @@
 import { OfflineAminoSigner, OfflineDirectSigner } from '@keplr-wallet/types';
 
+import {
+  SigningStargateClient,
+  defaultRegistryTypes,
+  StdFee,
+  DeliverTxResponse,
+} from '@cosmjs/stargate';
+import { TxRaw } from 'cosmjs-types/cosmos/tx/v1beta1/tx';
+import { GeneratedType, Registry } from '@cosmjs/proto-signing';
+import { MsgClaim } from './passage/msg_claim';
+import { Msg } from '../types/types';
+
 declare let window: WalletWindow;
 
+export async function signAndBroadcastAmino(
+  msgs: Msg[],
+  fee: number,
+  chainID: string,
+  rpcURL: string,
+  memo: string = ''
+): Promise<DeliverTxResponse> {
+  const result = await getWalletAmino(chainID);
+  const wallet = result[0];
```

Comment:
If the result outcome is undefined, can you explain how it will be dealt with?

Reply:
We won't be calling the function unless the wallet is connected, so it is almost guaranteed to return valid data